lib/tizen_plugins.dart

```diff
@@ -273,10 +274,11 @@ Future<void> injectTizenPlugins(FlutterProject project) async {
   if (tizenProject.existsSync()) {
     final List<TizenPlugin> nativePlugins =
         await findTizenPlugins(project, nativeOnly: true);
-    _writeCppPluginRegistrant(tizenProject.managedDirectory, nativePlugins);
-    _writeCsharpPluginRegistrant(tizenProject.managedDirectory, nativePlugins);
+    _writeTizenPluginRegistrant(tizenProject, nativePlugins);
```

Comment:
Tizen plugin can refer to either Dart or native plugin. Let's be more specific.

```suggestion
    _writeNativePluginRegistrant(tizenProject, nativePlugins);
```

Reply:
I agree, but I wanted to sync same style with other platforms in `flutter_plugins.dart`. It will  help us refer to other platforms' implementation later. And it implies that `writingXXXXPluginRegistrant` method name is for only native plugins.

Reply:
Ah great. That makes sense too.

Reply:
Thank you ❤️